Sources/Moya/Image.swift

```diff
@@ -7,4 +7,4 @@
 #endif
 
 /// An alias for the SDK's image type.
-public typealias Image = ImageType
+public typealias PlatformImage = ImageType
```

Comment:
We could also choose for something like `MoyaImage` but thought I'd go for a more generic name instead.

Reply:
Hey Bas, just an idea we could do something like this 
```suggestion
@available(*, deprecated, message: "Use `PlatformImage` instead")
public typealias Image = ImageType

public typealias PlatformImage = ImageType
```
That way it would be not a breaking change, so users can disambiguate between SwiftUI image right away without need to wait a major version :)

Reply:
Ooooh neat!